app/src/main/java/org/mozilla/focus/settings/privacy/PrivacySecuritySettingsFragment.kt

```diff
@@ -41,6 +43,12 @@ class PrivacySecuritySettingsFragment :
         ) {
             preferenceScreen.removePreference(biometricPreference)
         }
+        if (!Features.SHOULD_SHOW_TOOLTIP_FOR_PRIVACY_SECURITY_SETTINGS_SCREEN ||
+            !requireContext().settings.shouldShowPrivacySecuritySettingsToolTip ||
+            !WhatsNew.shouldHighlightWhatsNew(requireContext())
```

Comment:
I think we do not need this check. 
The display conditions described [here](https://github.com/mozilla-mobile/focus-android/issues/6211#issuecomment-1016067914) are covered without it, as the app needs an update to get the new value of the feature flag. 
In turn, if the user does not click the close button, based on UX specs, the CFR should continue to be displayed, even after some time has passed and when the `shouldHighlightWhatsNew` would return false.

Reply:
Done